portal/templates/portal/admin/import/import.html

```diff
@@ -0,0 +1,40 @@
+{% extends "portal/base.html" %}
+
+{% block content %}
+<section class="section">
+    <div class="container">
+        <div class="columns">
+            <div class="column is-3">
+                {% include "./_menu.html" %}
+            </div>
+
+            <div class="column">
+                <h1 class="title">{{ title }}</h1>
+
+                <div class="content">
+                    <p>The CSV should be exported from {{ source }} and is expected to contain the following columns (order doesn't matter, exact name does):</p>
+                    <ul>
+                        {% for col in expected_columns %}
+                        <li><code>{{ col }}</code></li>
+                        {% endfor %}
+                    </ul>
+                    <p>
+                        Importing is <strong>imdepotent</strong>, so you can upload the same file (or the same file with later additions)
```

Comment:
can you replace with something from https://thesaurus.com

Reply:
what

Reply:
imdepotent is too hard a word, pick something easier

Reply:
My reasoning including it here is that it's the correct, technical term, and I immediately define it in the rest of the sentence. Is it not clear that it is being defined? I think there's value in teaching the word here, what do you think? I'm not that attached to it. I'll merge in the meantime.